seed/serializers/meter_readings.py

```diff
@@ -4,23 +4,39 @@
 :copyright (c) 2014 - 2022, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.
 :author
 """
+from django.core.exceptions import ValidationError
+from django.db.utils import IntegrityError
 from rest_framework import serializers
 
 from seed.models import MeterReading
 
 
+class MeterReadingBulkCreateUpdateSerializer(serializers.ListSerializer):
+    def create(self, validated_data):
+        print(validated_data)
+        meter_data = [MeterReading(**item) for item in validated_data]
+
+        try:
+            result = MeterReading.objects.bulk_create(meter_data)
+        except IntegrityError as e:
+            raise ValidationError(e)
+
+        return result
```

Comment:
I don't think I understand why we need both serializes... other seralizers seem able to hand bulk and singular input.

```
FilterGroupSerializer(filter_groups, many=True).data
```

Reply:
The FilterGroupSerializer won't leverage the `bulk_create` db method. That has to happen by swapping out the list_serializer. The item you references if for printing/viewing the data, that works fine.